modules/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/virt-using-skip-node.adoc
+//
+
+[id="virt-using-skip-node_{context}"]
+= Using skip node annotation
+
```

Comment:
This needs a `.Procedure` heading.

For more information about writing procedures, see the [Red Hat modular docs reference guide](https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules) and the [procedure template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc).